test/built-ins/Promise/allSettled/reject-element-function-name.js

```diff
@@ -30,3 +30,4 @@ NotPromise.resolve = function(v) {
 Promise.allSettled.call(NotPromise, [thenable]);
 
 assert.sameValue(Object.prototype.hasOwnProperty.call(rejectElementFunction, 'name'), false);
+assert.sameValue(rejectElementFunction.name, '');
```

Comment:
I'm caught from another topic, but if I remember correctly we've had some discussion on anonymous function names in the last TC39 meeting. 

cc @gsathya, mind to remember me if something has changed there?

Reply:
The recent change I presented was to make the `name` property an `own` named property on the anonymous function. These tests just seem to be testing that the value of the property is an empty string, not the actual position of the property.

LGTM